rasa_nlu/data_router.py

```diff
@@ -176,8 +176,9 @@ def parse(self, data):
 
         model = self.model_store[alias]
         response = model.parse(data['text'])
+        log={"user_input": response, "model": alias, "timestamp": datetime.datetime.now().strftime('%Y-%m-%dT%H:%M:%S')}
```

Comment:
Since this is only used within the `if` block, lets move it there. Furthermore, I'd like to go with the default string format using `.isoformat()` instead of `.strftime`. If you don't mind, let's call it `time` or `datetime` or something like that, `timestamp` usually indicates a seconds / milliseconds time since 1970.